src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {

```

Comment:
The class `FruitTransaction` is missing access modifiers for its fields. Encapsulation is a key principle in object-oriented programming, and fields should be private to prevent direct access from outside the class. Consider making `operation`, `fruit`, and `quantity` private and providing setters if necessary for manipulation outside the class.